### PR TITLE
Update view column UI

### PR DESCRIPTION
**Pull request by adrinr**
## Description
Update grid's column picker. By changing the current simple toggle for a more complex one. This will allow us to add extra configurations (readonly columns, for example).

| Before | After |
|--------|--------|
| image | image |
| image | image |
| image | image | 

## Addresses
- [BUDI-8281 - [Readonly views] Update view column UI](https://linear.app/budibase/issue/BUDI-8281/[readonly-views]-update-view-column-ui)

## Launchcontrol
Updating grid's column picker.

### CODE REVIEW

File: packages/bbui/src/ActionButton/ActionButton.svelte [hunk 1]
```diff
@@ -109,19 +110,22 @@
     background: var(--spectrum-global-color-gray-300);
     border-color: var(--spectrum-global-color-gray-500);
   }
-  .noPadding {
```
Comment (adrinr) [inline packages/bbui/src/ActionButton/ActionButton.svelte:112]: This order is changed, otherwise `noPadding` and `quiet` was not working as expected

**Review by mike12345567 — APPROVED** [review]
Very nice - good idea to create the action button group!

**Review by aptkingston — COMMENTED** [review]
Couple of small suggested changes but nothing major. Nice work 👌

File: packages/frontend-core/src/components/grid/controls/ToggleActionButtonGroup.svelte [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  import { ActionButton } from "@budibase/bbui"
+
+  export let value
+  export let options
+  export let disabled
+</script>
+
+<div class="permissionPicker">
+  {#each options as option}
+    <ActionButton
```
[the remark this answers is not in the document]
Comment (adrinr) [reply packages/frontend-core/src/components/grid/controls/ToggleActionButtonGroup.svelte:15]: I did not know we had this component! This adjusts to the actual requested design (I thought it was just one of the things that we wanted to change eventually across the platform)

**Review by aptkingston — APPROVED** [review]

**Pull request merged**